src/lib_ccx/ccx_decoders_isdb.c

```diff
@@ -1329,13 +1327,9 @@ int isdb_parse_data_group(void *codec_ctx,const uint8_t *buf, struct cc_subtitle
 	ISDBSubContext *ctx = codec_ctx;
 	const uint8_t *buf_pivot = buf;
 	int id = (*buf >> 2);
-	int version = (*buf & 2);
-	int link_number = 0;
-	int last_link_number = 0;
 	int group_size = 0;
```

Comment:
Let's leave these ones, at least the version one. We might not be using in code, but it's useful when debugging - inspecting the variable and checking that the value is sane is a good way to see if things are going OK.